web/src/components/admin/connectors/AdminSidebar.tsx

```diff
@@ -93,6 +94,16 @@ export function AdminSidebar({ collections }: { collections: Collection[] }) {
           </div>
         ))}
       </nav>
+      {combinedSettings.webVersion && (
+        <div
+          className="flex  flex-col mt-4 items-center justify-center w-full"
+          key={"danswerVersion"}
+        >
+          <h2 className="text-sm text-text w-52 font-medium pb-2">
+            Danswer version {combinedSettings.webVersion}
```

Comment:
nit: prefer smaller (you shouldn't be drawn to it unless you're explicitly looking for it imo) + larger spacing from the rest (same reasoning) + some separation between Danswer version and the version. Maybe something like:

Danswer Version: <pre>{combinedSettings.webVersion}</pre>

or

<b>Danswer Version:</b> {combinedSettings.webVersion}